scripts/install_jupyter.sh

```diff
@@ -36,6 +36,15 @@ install2.r --error --skipmissing --skipinstalled -n "$NCPUS" remotes
 R --quiet -e 'remotes::install_github("IRkernel/IRkernel@*release")'
 R --quiet -e 'IRkernel::installspec(user = FALSE)'
 
+## Install texlive
+/rocker_scripts/install_texlive.sh
+# Install tex packages needed for Jupyter's nbconvert to work correctly & convert to PDF
+# Sourced from https://github.com/jupyter/nbconvert/issues/1328
+tlmgr install adjustbox caption collectbox enumitem environ eurosym etoolbox jknapltx parskip \
+    pdfcol pgf rsfs tcolorbox titling trimspaces ucs ulem upquote \
+    ltxcmds infwarerr iftex kvoptions kvsetkeys float geometry amsmath fontspec \
+    unicode-math fancyvrb grffile hyperref booktabs soul ec
```

Comment:
Could you please make this optional?
If my understanding is correct, the expected behavior of this script is to set jupyter, and this should not be done by default.

And check if `tlmgr` is already installed before exec `install_texlive.sh`.

Reply:
I'll check for tlmgr before installing!

Reply:
I'd consider 'does export to PDF from jupyter work?' to be part of 'set jupyter'. In the jupyter community we have found it important enough that it's part of the minimal notebook package in the jupyter equivalent of rocker (https://github.com/jupyter/docker-stacks/blob/main/docs/using/selecting.md#jupyterminimal-notebook). In the same way I think RStudio users would expect to knit, I think Jupyter users expect to be able to print to PDF. So I'd strongly hope this can be kept for the default installation, as there's only one rocker image that has Jupyter in it.

Reply:
I've made the install conditional now

Reply:
Thanks for looking into that.
I haven't personally used this, so I'll leave the final judgment to @cboettig.
(However, I personally would like to see this become an opt-in feature.)

Also, since this is a big change, could you please update the NEWS?

Reply:
I think this makes sense as the default given the expectations from the Jupyter community.  Definitely agree with mentioning this in NEWS.  Otherwise looks good to me!

Reply:
Thanks @cboettig and @eitsupi. I've added an entry to the NEWS file.